lib/templates.ts

```diff
@@ -1395,6 +1398,71 @@ export async function deleteTemplate(formID: string): Promise<FormRecord | null>
   return _parseTemplate(templateMarkedAsDeleted);
 }
 
+/**
+ * Restores a form template from the archived state.
+ * @param formID ID of the form template
+ * @returns A boolean status if operation is sucessful
+ */
+export async function restoreTemplate(formID: string): Promise<FormRecord | null> {
+  const { user } = await authorization.canDeleteForm(formID).catch((e) => {
+    logEvent(
+      e.user.id,
+      { type: "Form", id: formID },
+      "AccessDenied",
+      "Attempted to unarchive Form"
+    );
+    throw e;
+  });
+
+  // Check if the form is draft or not.
```

Comment:
I may be wrong but I feel like this comment does not reflect what is actually being achieved below. Especially considering the thrown error is `TemplateNotFoundError`.